sirepo/template/radia.py

```diff
@@ -800,21 +795,22 @@ def _parse_input_file_arg_str(s):
 
 
 def _prep_new_sim(data):
-    data.models.geometry.name = data.models.simulation.name
+    data.models.geometryReport.name = data.models.simulation.name
 
 
 def _read_h5_path(sim_id, filename, h5path, run_dir=_GEOM_DIR):
     try:
-        with h5py.File(_get_res_file(sim_id, filename, run_dir=run_dir), 'r') as hf:
-            return template_common.h5_to_dict(hf, path=h5path)
+        p = _get_sim_file(sim_id, filename, run_dir=run_dir)
+        with h5py.File(p, 'r') as f:
+            return template_common.h5_to_dict(f, path=h5path)
     except IOError as e:
         if pkio.exception_is_not_found(e):
-            pkdc(f'{filename} not found in {run_dir}')
+            pkdlog(f'file {filename} not found in {p}')
             # need to generate file
             return None
     except KeyError:
```

Comment:
This is too course of an error. I don't see where `KeyError` is being thrown, but it might be thrown in any of the functions. Better to wrap this deeper in the code, I think.

Reply:
7679250

Added ```NoH5PathError``` which seems general enough to put in ```template_common```